packages/loader/container-loader/src/container.ts

```diff
@@ -1769,4 +1781,60 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     private logContainerError(warning: ContainerWarning) {
         this.logger.sendErrorEvent({ eventName: "ContainerWarning" }, warning);
     }
+
+    private async loadContainerCopy(
+        // back-compat 0.35 stop passing loader post 0.35 support
+        loader: Loader,
+        urlResolver: IUrlResolver,
+        baseRequestUrl: string | undefined,
+        additionalHeaders: IRequestHeader,
+    ): Promise<IContainer> {
+        if (baseRequestUrl === undefined) {
+            throw new Error("Base request is not provided");
+        }
+
+        // TODO #4912 eventually we may wish to spawn an execution context from which to run this
+        const containerRequest: IRequest = {
+            headers: {
+                ...additionalHeaders,
+                [LoaderHeader.cache]: false,
+                [LoaderHeader.reconnect]: false,
+            },
+            url: baseRequestUrl,
+        };
+
+        const resolvedAsFluid = await urlResolver.resolve(containerRequest);
+        ensureFluidResolvedUrl(resolvedAsFluid);
+
+        // Parse URL into data stores
+        const parsed = parseUrl(resolvedAsFluid.url);
+        if (parsed === undefined) {
+            return Promise.reject(new Error(`Invalid URL ${resolvedAsFluid.url}`));
+        }
+
+        // parseUrl's id is expected to be of format "tenantId/docId"
+        const [, docId] = parsed.id.split("/");
+
+        const container = await Container.load(
+            docId,
+            loader,
+            containerRequest,
+            resolvedAsFluid);
+
+        const fromSequenceNumber = additionalHeaders[LoaderHeader.sequenceNumber] ?? -1;
```

Comment:
And here probably either take the `sequenceNumber` directly as a param, or else return the `Container` as-loaded and let the caller wait on the `sequenceNumber` themselves.  I'm interested in trying to keep `LoaderHeader` out of `Container` :)